tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -20,3 +24,18 @@ class TestOnDemandCluster(tests.test_resources.test_clusters.test_cluster.TestCl
             "multinode_cpu_cluster",
         ]
     }
+
+    @pytest.mark.level("minimal")
+    def test_rh_status_errors(self, cluster):
```

Comment:
This does doesn't need to be limited to an ondemand cluster, thoughts on making it a local test and moving into "test_cluster"? is there any reason why using a local docker cluster wouldn't be enough here?